test/uk/gov/hmrc/alcoholdutyaccount/models/subscription/AdrSubscriptionSummarySpec.scala

```diff
@@ -51,18 +72,46 @@ class AdrSubscriptionSummarySpec extends SpecBase {
     Seq(
       (hods.Beer, Set(AlcoholRegime.Beer)),
       (hods.CiderOrPerry, Set(AlcoholRegime.Cider, AlcoholRegime.OtherFermentedProduct)),
-      (hods.WineAndOtherFermentedProduct, Set(AlcoholRegime.Wine, AlcoholRegime.OtherFermentedProduct)),
-      (hods.Spirits, Set(AlcoholRegime.Spirits))
+      (hods.Wine, Set(AlcoholRegime.Wine, AlcoholRegime.OtherFermentedProduct)),
+      (hods.Spirits, Set(AlcoholRegime.Spirits)),
+      (hods.OtherFermentedProduct, Set(AlcoholRegime.OtherFermentedProduct))
```

Comment:
Since the toggle is off, shouldn't we expect the set of expectedRegimes here to be empty?

Reply:
No, if you check the logic, it initially won't get a hods.OtherFermentedProduct so this will be set either via Cider or Wine. However, when ETMP changes, it'll start sending this, and we need to still work until we change the feature toggle, thus OFP might be the only approval, so we'll need to handle it in the old way. The only difference between old and new programmatically is old also sets OFP on Cider and Wine.

Reply:
Not sure what you mean on this one. The API should make all regimes individually selected, we need to check they are translated from the API (hods) enums to AlcoholRegime which is our internal one.

Reply:
I think I was confused by OFP being an approved regime when OFP comes back from the API, since it was feature toggled off. Looking back now, it's covered by your comment here (https://github.com/hmrc/alcohol-duty-account/pull/32/files#diff-f99e4646866e7ed8d531976b7794e37a8d85235f4a3fd62538550b2ff6da570fR71)